snap/snapcraft.yaml

```
@@ -0,0 +1,24 @@
+name: sabnzbd
+version: git
+summary: SABnzbd
+description: The automated Usenet download tool
+grade: stable
+confinement: strict
+
+apps:
+  sabnzbd:
+    command: opt/sabnzbd/SABnzbd.py -f $SNAP_COMMON
+    daemon: simple
+    plugs: [network, network-bind]
```

```suggestion
    plugs:
    - network
    - network-bind
    - removable-media
```

You probably want to include the `removable-media` plug because it is likely that a user will want to store their downloads in a location other than `$SNAP_DATA` or `$SNAP_COMMON`.